src/containerapp/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 0.3.26
 +++++++
 * 'az containerapp exec': fix bugs for consumption workload based environment
+* Add CLI support for ingress sticky-sessions'
```

Comment:
would it be better to add the exact command group that we have added for ingress sticky sessions here? 'containerapp ingress sticky-sessions'

Reply:
also I believe this should be added in an Upcoming section instead of under the 0.3.26 notes

Reply:
what do you mean `command group` ? can you elaborate a little more ?

Reply:
the command that has been added for [sticky sessions](https://github.com/Azure/azure-cli-extensions/blob/81e5104f8aeb2267b5f3c5cac1e90fb0c45a05c2/src/containerapp/azext_containerapp/commands.py#L125).

Reply:
done